src/types/CTypeMetedata.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @module TypeInterfaces/CType
+ */
+/**
+ * Dummy comment needed for correct doc display, do not remove.
+ */
+export default interface ICTypeMetadata {
+  metadata: IMetadata
```

Comment:
@maudnals I had to remove the id and schema, because of the demo client. It didn't allow for them.  without changing more of the input data structure in the backend to make it work.